AppCenter/AppCenterTests/MSAppCenterIngestionTests.m

```diff
@@ -13,6 +13,8 @@
 static NSTimeInterval const kMSTestTimeout = 5.0;
 static NSString *const kMSBaseUrl = @"https://test.com";
 static NSString *const kMSTestAppSecret = @"TestAppSecret";
+static NSString *const kMSAuthorizationHeaderKey = @"Authorization";
```

Comment:
Can you use the same ones as in `MSAppCenterIngestion.m` to avoid duplication?

Reply:
Can probably reuse the ingestion ones.